diff --git a/e2e/tests/oai/002.sh b/e2e/tests/oai/002.sh
index a334090..6e7f6e0 100755
--- a/e2e/tests/oai/002.sh
+++ b/e2e/tests/oai/002.sh
@@ -29,7 +29,7 @@ source "${LIBDIR}/kpt.sh"
 
 k8s_apply "$TESTDIR/002-oai-operators.yaml"
 
-for pkgvar in common-core-database cp-operators up-operators ran-operator-edge-oai-ran-operator ran-operator-regional-oai-ran-operator; do
+for pkgvar in common-core-database cp-operators up-operators ran-operator-edge ran-operator-regional; do
     k8s_wait_ready "packagevariant" "oai-$pkgvar"
 done
 kpt_wait_pkg "core" "database"
